index.html

```diff
@@ -0,0 +1,260 @@
+                <div class="SpeakersCont" id="speakersContainer"></div>
+                <div class="SeeMoreButon">
+                    <a href="#" class="SeeMoreBtn BtnR" id="seeMoreBtn">
+                        See More
+                    </a>
+                </div>
+            </div>
+        </section>
+        <section class="main1" id="main1">
+            <div class="firsttemplate1">
+                <div class="info1">
+                    <div class="title11">
+                        <h2 class="title2">The E-Commerce </br>
+                            Pioneers Max 2023
+                        </h2>
+                    </div>
+                    <div class="squareInfo1">
+                        <h4 class="resumetitle1">
+                            The Global Summit brings together the community
+                            of experts, academics, and activists who comprise 
+                            the Creative Commons affiliate network in a 
+                            different country every two years. This year, 
+                            we hope to expand our invitation list including 
+                            organizations and individuals.
+                        </h4>
+                    </div>
+                    <div class="dateTitle1" id="dateTitle1">
+                        Please contact us per Email for nay further questions about CC Global Summit 2015!
+                    </div>
+                    <div class="siteDir1">ccsummit2015@anaxtasia.com</div>
+                    <div class="ContainerSectionSpeakers">
+                        <div class="SectionSpeakT1">
+                            Creative Commons Global Summit 2015 Logo
+                        </div>
+                        <div class="RedLine"></div>
+                        <div class="dateTitle11">
+                            The logo of CC Global Summit 2015 was 
+                            decided through the logo competition 
+                            from 8, June to 7, July.
+                        </div>
+                        <img id="anaxtasia" src="./images/anaxtasia.jpg" alt="anax"/>
+                            <div class="SectionSpeakT1">
+                                See the past CC Global Summits
+                            </div>                          
+                            <div class="RedLine"></div>
+                            <div class="penul" id="penul">a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</div>
+                            <div class="penul1">
+                                <img id="anaxtasia" src="./images/about_2011 1.png" alt="anax"/>
+                                <img id="anaxtasia" src="./images/about_2013 1.png" alt="anax"/>
+                            </div> 
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Good job displaying this section horizontally as requested, however kindly add some spacing to the `divs` so as to match the given design

Your design
![image](https://github.com/jovasal/Capstone/assets/81932478/4bc3f35b-f371-4054-aa89-03d9ddcb9c76)

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/058d1878-e376-4eb5-bf32-3f3ebb3d0c62)

#### Note that the About page has no `Partner` section in the given desktop design, but rather has it in the mobile version.

Reply:
ok. done
![image](https://github.com/jovasal/Capstone/assets/35693330/a88a89a4-1f68-4094-beef-60105dc22888)